diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 595e497..4dd8366 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -13,6 +13,8 @@ require_once __DIR__.'/../../vendor/phpunit/phpunit/src/Framework/Assert/Functio
  */
 class FeatureContext extends RawMinkContext implements Context, SnippetAcceptingContext
 {
+    private static $container;
+
     /**
      * Initializes context.
      *
@@ -25,11 +27,31 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
     }
 
     /**
-     * @Given there is an admin user :arg1 with password :arg2
+     * @BeforeSuite
+     */
+    public static function bootstrapSymfony()
+    {
+        require_once __DIR__.'/../../app/autoload.php';
+        require_once __DIR__.'/../../app/AppKernel.php';
+
+        $kernel = new AppKernel('test', true);
+        $kernel->boot();
+        self::$container = $kernel->getContainer();
+    }
+
+    /**
+     * @Given there is an admin user :username with password :password
      */
-    public function thereIsAnAdminUserWithPassword($arg1, $arg2)
+    public function thereIsAnAdminUserWithPassword($username, $password)
     {
-        throw new PendingException();
+        $user = new \AppBundle\Entity\User();
+        $user->setUsername($username);
+        $user->setPlainPassword($password);
+        $user->setRoles(array('ROLE_ADMIN'));
+
+        $em = self::$container->get('doctrine')->getManager();
+        $em->persist($user);
+        $em->flush();
     }
 
     /**
diff --git a/features/web/authentication.feature b/features/web/authentication.feature
index f5634d5..4c76748 100644
--- a/features/web/authentication.feature
+++ b/features/web/authentication.feature
@@ -4,10 +4,10 @@ Feature: Authentication
   I need to be able to login and logout
 
   Scenario: Logging in
-    Given there is an admin user "admin" with password "admin"
+    Given there is an admin user "admin2" with password "admin"
     And I am on "/"
     When I follow "Login"
-    And I fill in "Username" with "admin"
+    And I fill in "Username" with "admin2"
     And I fill in "Password" with "admin"
     And I press "Login"
     Then I should see "Logout"
