diff --git a/composer.lock b/composer.lock
index 032a645..bec0d20 100644
--- a/composer.lock
+++ b/composer.lock
@@ -4,8 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "hash": "21dd74a1c4eb1f044398c2cfbc4667ae",
-    "content-hash": "519a36ab4a9691b442ad1650d1a35f43",
+    "hash": "3477d543c87895ece69ac73ccf2d8161",
     "packages": [
         {
             "name": "behat/behat",
diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index ec12eb6..184135e 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -5,6 +5,7 @@ use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\Gherkin\Node\PyStringNode;
 use Behat\Gherkin\Node\TableNode;
 use Behat\MinkExtension\Context\RawMinkContext;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
 
 require_once __DIR__.'/../../vendor/phpunit/phpunit/src/Framework/Assert/Functions.php';
 
@@ -31,9 +32,8 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
      */
     public function clearData()
     {
-        $em = $this->getContainer()->get('doctrine')->getManager();
-        $em->createQuery('DELETE FROM AppBundle:Product')->execute();
-        $em->createQuery('DELETE FROM AppBundle:User')->execute();
+        $purger = new ORMPurger($this->getContainer()->get('doctrine')->getManager());
+        $purger->purge();
     }
 
     /**
