diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index ed88c19..b80ddcd 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -106,7 +106,7 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
             $product->setPrice(rand(10, 1000));
             $product->setDescription('lorem');
 
-            if ($row['is published'] == 'yes') {
+            if (isset($row['is published']) && $row['is published'] == 'yes') {
                 $product->setIsPublished(true);
             }
 
