diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 38494a3..cd7e3ff 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -110,7 +110,7 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
             $product->setPrice(rand(10, 1000));
             $product->setDescription('lorem');
 
-            if ($row['is published'] == 'yes') {
+            if (isset($row['is published']) && $row['is published'] == 'yes') {
                 $product->setIsPublished(true);
             }
 
