diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 3071421..399058f 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -132,7 +132,10 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
      */
     public function iWaitForTheModalToLoad()
     {
-        $this->getSession()->wait(5000);
+        $this->getSession()->wait(
+            5000,
+            "$('.modal:visible').length > 0"
+        );
     }
 
     /**
