diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 565508a..59e89d6 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -136,7 +136,10 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
      */
     public function iWaitForTheModalToLoad()
     {
-        $this->getSession()->wait(5000);
+        $this->getSession()->wait(
+            5000,
+            "$('.modal:visible').length > 0"
+        );
     }
 
     /**
