diff --git a/features/web/authentication.feature b/features/web/authentication.feature
index 5c834bc..1e63803 100644
--- a/features/web/authentication.feature
+++ b/features/web/authentication.feature
@@ -2,3 +2,11 @@ Feature: Authentication
   In order to gain access to the site management area
   As an admin
   I need to be able to login and logout
+
+  Scenario: Logging in
+    Given I am on "/"
+    When I follow "Login"
+    And I fill in "Username" with "admin"
+    And I fill in "Password" with "adminpass"
+    And I press "Login"
+    Then I should see "Logout"
