diff --git a/features/web/authentication.feature b/features/web/authentication.feature
index a32cd72..7d2d854 100644
--- a/features/web/authentication.feature
+++ b/features/web/authentication.feature
@@ -7,7 +7,6 @@ Feature: Authentication
     Given I am on "/"
     When I follow "Login"
     And I fill in "Username" with "admin"
-    And I fill in "Password" with "adminpass"
+    And I fill in "Password" with "admin"
     And I press "Login"
-    Then show last response
     Then I should see "Logout"
