diff --git a/mink.php b/mink.php
index 6f928d1..73ca36b 100644
--- a/mink.php
+++ b/mink.php
@@ -21,3 +21,9 @@ echo "Current URL: ". $session->getCurrentUrl() . "\n";
 $page = $session->getPage();
 
 echo "First 75 chars: ".substr($page->getText() , 0, 75) . "\n";
+
+// Important object #4 NodeElement
+$header = $page->find('css', '.wds-community-header__sitename a');
+
+echo "The wiki site name is: ".$header->getText()."\n";
+
