diff --git a/mink.php b/mink.php
index 8d558a2..df5a88f 100644
--- a/mink.php
+++ b/mink.php
@@ -30,5 +30,13 @@ echo "The wiki site name is: ".$header->getText()."\n";
 $subNav = $page->find('css', '.wds-tabs');
 $linkEl = $subNav->find('css', 'li a');
 
-echo "The link text is: ". $linkEl->getText() . "\n";
-
+$selectorsHandler = $session->getSelectorsHandler();
+$linkEl = $page->find(
+    'named',
+    array(
+        'link',
+        $selectorsHandler->xpathLiteral('Books')
+    )
+);
+
+echo "The link href is: ". $linkEl->getAttribute('href') . "\n";
