diff --git a/mink.php b/mink.php
index d396efc..6ad17f4 100644
--- a/mink.php
+++ b/mink.php
@@ -3,10 +3,12 @@
 require __DIR__.'/vendor/autoload.php';
 
 use Behat\Mink\Driver\GoutteDriver;
+use Behat\Mink\Driver\Selenium2Driver;
 use Behat\Mink\Session;
 
 // Important object #1
-$driver = new GoutteDriver();
+//$driver = new GoutteDriver();
+$driver = new Selenium2Driver();
 
 // Important object #2
 $session = new Session($driver);
@@ -14,7 +16,7 @@ $session = new Session($driver);
 $session->start();
 $session->visit('http://jurassicpark.wikia.com');
 
-echo "Status code: ". $session->getStatusCode() . "\n";
+//echo "Status code: ". $session->getStatusCode() . "\n";
 echo "Current URL: ". $session->getCurrentUrl() . "\n";
 
 // Important object #3 DocumentElement
@@ -36,3 +38,5 @@ echo "The link href is: ". $linkEl->getAttribute('href') . "\n";
 
 $linkEl->click();
 echo "Page URL after click: ". $session->getCurrentUrl() . "\n";
+
+$session->stop();
\ No newline at end of file
