diff --git a/mink.php b/mink.php
index 19616e7..f28209a 100644
--- a/mink.php
+++ b/mink.php
@@ -28,5 +28,7 @@ $header = $page->find('css', '.WikiHeader .WikiNav h1');
 echo "The wiki nav text is: ".$header->getText()."\n";
 
 $subNav = $page->find('css', '.subnav-2');
-var_dump($subNav->getHtml());
+$linkEl = $subNav->find('css', 'li a');
+
+echo "The link text is: ". $linkEl->getText() . "\n";
 
