diff --git a/behat.yml b/behat.yml
index ccef6ba..48194d1 100644
--- a/behat.yml
+++ b/behat.yml
@@ -3,8 +3,13 @@ default:
         default:
             contexts:
                 - FeatureContext
-                - CommandLineProcessContext
                 - Behat\MinkExtension\Context\MinkContext
+            paths: [ %paths.base%/features/web ]
+        commands:
+            contexts:
+                - CommandLineProcessContext
+            paths: [ %paths.base%/features/commands ]
+
     extensions:
         Behat\MinkExtension:
             base_url: http://localhost:8000
diff --git a/features/ls.feature b/features/commands/ls.feature
similarity index 100%
rename from features/ls.feature
rename to features/commands/ls.feature
diff --git a/features/authentication.feature b/features/web/authentication.feature
similarity index 100%
rename from features/authentication.feature
rename to features/web/authentication.feature
diff --git a/features/fence_api.feature b/features/web/fence_api.feature
similarity index 100%
rename from features/fence_api.feature
rename to features/web/fence_api.feature
diff --git a/features/product_admin.feature b/features/web/product_admin.feature
similarity index 100%
rename from features/product_admin.feature
rename to features/web/product_admin.feature
diff --git a/features/search.feature b/features/web/search.feature
similarity index 100%
rename from features/search.feature
rename to features/web/search.feature
