diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 21a8f73..86ea914 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -41,6 +41,14 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
     }
 
     /**
+     * @BeforeScenario @fixtures
+     */
+    public function loadFixtures()
+    {
+        var_dump('GO!');
+    }
+
+    /**
      * @Given there is an admin user :username with password :password
      */
     public function thereIsAUserWithPassword($username, $password)
diff --git a/features/web/search.feature b/features/web/search.feature
index a1373d0..469a811 100644
--- a/features/web/search.feature
+++ b/features/web/search.feature
@@ -6,6 +6,7 @@ Feature: Search
   Background:
     Given I am on "/"
 
+  @fixtures
   Scenario Outline: Search for a product
     When I fill in the search box with "<term>"
     And I press the search button
