diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 3f76945..a217a23 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -1,5 +1,6 @@
 <?php
 
+use AppBundle\Entity\Product;
 use Behat\Behat\Context\Context;
 use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\Gherkin\Node\PyStringNode;
@@ -73,6 +74,42 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
         $button->press();
     }
 
+    /**
+     * @Given there are :count products
+     */
+    public function thereAreProducts($count)
+    {
+        for ($i = 0; $i < $count; $i++) {
+            $product = new Product();
+            $product->setName('Product '.$i);
+            $product->setPrice(rand(10, 1000));
+            $product->setDescription('lorem');
+
+            $this->getEntityManager()->persist($product);
+        }
+
+        $this->getEntityManager()->flush();
+    }
+
+    /**
+     * @When I click :linkName
+     */
+    public function iClick($linkName)
+    {
+        $this->getPage()->clickLink($linkName);
+    }
+
+    /**
+     * @Then I should see :count products
+     */
+    public function iShouldSeeProducts($count)
+    {
+        $table = $this->getPage()->find('css', 'table.table');
+        assertNotNull($table, 'Cannot find a table!');
+
+        assertCount(intval($count), $table->findAll('css', 'tbody tr'));
+    }
+
     /**
      * @return \Behat\Mink\Element\DocumentElement
      */
@@ -80,4 +117,12 @@ class FeatureContext extends RawMinkContext implements Context, SnippetAccepting
     {
         return $this->getSession()->getPage();
     }
+
+    /**
+     * @return \Doctrine\ORM\EntityManager
+     */
+    private function getEntityManager()
+    {
+        return $this->getContainer()->get('doctrine.orm.entity_manager');
+    }
 }
diff --git a/features/web/product_admin.feature b/features/web/product_admin.feature
index 1ce4516..6192938 100644
--- a/features/web/product_admin.feature
+++ b/features/web/product_admin.feature
@@ -7,6 +7,7 @@ Feature: Product admin panel
     Given there are 5 products
     And I am on "/admin"
     When I click "Products"
+    And print last response
     Then I should see 5 products
 
   Scenario: Add a new product
